src/test/kotlin/uk/gov/dluhc/printapi/rest/aed/SearchAedSummariesIntegrationTest.kt

```diff
@@ -0,0 +1,168 @@
+import uk.gov.dluhc.printapi.testsupport.bearerToken
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidApplicationReference
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidSourceReference
+import uk.gov.dluhc.printapi.testsupport.testdata.anotherValidEroId
+import uk.gov.dluhc.printapi.testsupport.testdata.entity.buildAedContactDetails
+import uk.gov.dluhc.printapi.testsupport.testdata.entity.buildAnonymousElectorDocument
+import uk.gov.dluhc.printapi.testsupport.testdata.getVCAnonymousAdminBearerToken
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildAedSearchSummaryApiFromAedEntity
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildElectoralRegistrationOfficeResponse
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildLocalAuthorityResponse
+import java.time.Instant
+import java.time.LocalDate
+
+internal class SearchAedSummariesIntegrationTest : IntegrationTest() {
+    companion object {
+        private const val SEARCH_SUMMARY_URI_TEMPLATE = "/eros/{ERO_ID}/anonymous-elector-documents/search"
+        private const val GSS_CODE = "W06000099"
+        private const val ANOTHER_GSS_CODE = "E06000123"
+    }
+
+    @Test
+    fun `should return forbidden given user with valid bearer token belonging to a different ero`() {
+        wireMockService.stubCognitoJwtIssuerResponse()
+        val userGroupEroId = anotherValidEroId(ERO_ID)
+
+        webTestClient.get()
+            .uri(SEARCH_SUMMARY_URI_TEMPLATE, ERO_ID)
+            .bearerToken(getVCAnonymousAdminBearerToken(eroId = userGroupEroId))
+            .contentType(APPLICATION_JSON)
+            .exchange()
+            .expectStatus()
+            .isForbidden
+    }
+
+    @Test
+    fun `should return empty list given no AEDs exist for an ERO`() {
+        // Given
+        val eroResponse = buildElectoralRegistrationOfficeResponse(
+            id = ERO_ID,
+            localAuthorities = listOf(buildLocalAuthorityResponse(gssCode = GSS_CODE))
+        )
+        wireMockService.stubCognitoJwtIssuerResponse()
+        wireMockService.stubEroManagementGetEroByEroId(eroResponse, ERO_ID)
+
+        // When
+        val response = webTestClient.get()
+            .uri(SEARCH_SUMMARY_URI_TEMPLATE, ERO_ID)
+            .bearerToken(getVCAnonymousAdminBearerToken(eroId = ERO_ID))
+            .contentType(APPLICATION_JSON)
+            .exchange()
+            .expectStatus().isOk
+            .returnResult(AedSearchSummaryResponse::class.java)
+
+        // Then
+        val actual = response.responseBody.blockFirst()
+        assertThat(actual!!.results).isEmpty()
+    }
+
+    @Test
+    fun `should return summaries given AEDs exist for an ERO`() {
+        // Given
+        val eroResponse = buildElectoralRegistrationOfficeResponse(
+            id = ERO_ID,
+            localAuthorities = listOf(buildLocalAuthorityResponse(gssCode = GSS_CODE))
+        )
+        val currentDate = LocalDate.now()
```

Comment:
Minor - can you move these 4 lines (L73-L76) down a few lines please, so that the setting up the wiremock stuff is together? (ie. like you have it on L45-L50 above)